### PR TITLE
CLI: Repro refinements per feedback

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

Follow-up to @yannbf 's comments on discord

- [x] Reorder repro prompts
- [x] Add next steps instructions

Plus minor cleanups

## How to test

```
./lib/cli/bin/index.js repro
```

### CODE REVIEW

**Comment by gaetanmaisse** [comment]
I added a commit to keep only the logs relevant for the users and improve the formatting when there is an error, see screenshot.

Everything-is-ok case:
image

Error case:
image

--

Also, I have in mind to improve, in another PR, the logging in the whole SB CLI by:
 - Using `@storybook/node-logger` logger instead of the simple `console.xxx` we are using right now
 - Setting the default log level to `info` and add a `--verbose` option available for all SB CLI commands that will set the log level accordingly. And then move all the `log.debug` we currently have to `verbose` (especially the shell command we are running under the hood when bootstrapping a repro).

WDYT?

File: lib/cli/src/repro.ts [hunk 1]
```diff
@@ -93,15 +82,28 @@ export const repro = async ({
...
+
+  let selectedDirectory = outputDirectory;
+  if (!selectedDirectory) {
+    const { directory } = await prompts({
+      type: 'text',
+      message: 'Enter the output directory',
+      name: 'directory',
+    });
+    selectedDirectory = directory;
+    // if (fs.existsSync(selectedDirectory)) {
```
Comment (yannbf) [inline lib/cli/src/repro.ts:96]: Is this comment intentional?

File: lib/cli/src/repro.ts [hunk 2]
```diff
@@ -111,8 +113,19 @@ export const repro = async ({
...
     }
+
+    logger.info('');
+    logger.info('🎉 Your Storybook reproduction project is ready to use! Now:');
+    logger.info('');
+    logger.info(chalk.yellow(`cd ${selectedDirectory}`));
+    logger.info(chalk.yellow(`yarn storybook`));
+    logger.info('');
+    logger.info("Once you've recreated the problem you're experiencing,");
+    logger.info('publish to github and  paste the repo link in your issue.');
```
Comment (yannbf) [inline lib/cli/src/repro.ts:124]:
```suggestion
    logger.info('publish to github and paste the repo link in your issue.');
```

**Review by yannbf — APPROVED** [review]
Looks much better! Thanks for taking action on the feedback <3

**Comment by shilman** [comment]
@yannbf here you go ❤️ 

-bash_and_-bash

**Comment by shilman** [comment]
@gaetanmaisse I love your logging improvements. However, AFAIK all the `logger.x` commands prefix the line with the error level and there are some things we probably don't want that for. WDYT?

**Pull request merged**